### DISCUSSION TITLE
Sinclair coefficient

### DISCUSSION THREAD

**Discussion by PandaTheReal1** [jflamy/owlcms4#810] [General]
Hi
I have a question for the where the sinclair coefficient is calculated. I have a small competition, where i would like to compare men and women. Here we basically want to multiply the womens sinclair points with 1.39.
Is it possible to change this in the code, so all the womens sinclair points show as the updated one?

**Comment by jflamy** [comment]
No need to change any code.
You can change the Sinclair table by editing the `.properties` files in the `local/sinclair` folder in the installation directory.
You would change the values for the women to your adjusted values.

**Reply by PandaTheReal1** [reply]
Wow, okay. That seems easy enough, thank you!

**Reply by jflamy** [reply]
There are two Sinclair tables actually. You could leave one standard (the 2000 one is the one most people still use) and override the 2004.
If you do that, the Masters coefficients would still work -- they rely on the 2000 table.
